lit_parrot/speed_monitor.py

```diff
@@ -57,6 +57,19 @@
     "quadro rtx 5000": {"32-true": 11.2e12, "16-true": 89.2e12, "16-mixed": 89.2e12},
 }
 
+TPU_AVAILABLE_FLOPS={
+    # flop count for each TPU generation is the same for all precisions
+    # since bfloat16 precision is always used for performing matrix operations
+    # for more info: https://cloud.google.com/tpu/docs/bfloat16#choosing_bfloat16
+
+    # source: https://arxiv.org/pdf/1907.10701.pdf
+    "V2": 45e12,
+    # source: https://cloud.google.com/tpu/docs/system-architecture-tpu-vm#tpu_v3
+    "V3": 123e12,
```

Comment:
Shouldn't split the specific configs (v4-8, v4-128, ...)? As I had in https://github.com/Lightning-AI/lit-gpt/pull/172/files#diff-d83a788c6cc66e57022968bc902a9c15ff5e29f19e55f0f0aaadaae37f132cb8R59-R61